web/Services/LandlordService.cs

```diff
@@ -231,6 +239,36 @@ public async Task UnapproveLandlord(int landlordId)
         await _dbContext.SaveChangesAsync();
     }
 
+    public async Task<ILandlordService.DisableLandlordResult> DisOrEnableLandlord(int landlordId, string action)
+    {
+        var landlord = await GetLandlordIfExistsFromId(landlordId);
+        if (landlord is null)
+        {
+            return ILandlordService.DisableLandlordResult.ErrorLandlordNotFound;
+        }
+
+        if (action == "disable")
+        {
+            if (landlord.Disabled)
+            {
+                return ILandlordService.DisableLandlordResult.ErrorAlreadyDisabled;
+            }
+
+            landlord.Disabled = true;
+        }
+        else
+        {
+            if (!landlord.Disabled)
+            {
+                return ILandlordService.DisableLandlordResult.ErrorAlreadyDisabled;

```

Comment:
This error seems unintuitive to me - this is getting returned if someone tries to enable an already enabled landlord, right? So I wouldn't expect to see a line of code returning an error that says 'already _dis_ abled'

Reply:
Because the two methods are just the reverse of each other, they're sharing error statuses (with the message returned just substituting in 'disable' or 'enable' - I'll rename it to make it clearer.